.github/workflows/tests.yml

```diff
@@ -199,9 +202,11 @@ jobs:
         run: |
           python -m pytest --pyargs holospy
 
-      # - name: Run kikuchipy Test Suite
-      #   run: |
-      #     python -m pytest --pyargs kikuchipy
+      - name: Run kikuchipy Test Suite
+        if: ${{ always() }}
+        # run the tests headlessly
+        run: |
+          xvfb-run python -m pytest --pyargs kikuchipy -k "not test_spherical_pyvista and not test_not_allow_download_raises"
```

Comment:
Why is it necessary to skip these tests?  Please add a note to justify.
Usually we skip test when:
- temporally: an issue has been identified and it is fixed upstream and skipping test manually (for example here) is expected to be removed at some point
- workaround: there is a failure on CI which is not understood, for example it can be related to different blas implementation, etc. most likely relevant to a subtle bug or ill defined test.

I think that this is good practise to keep skipping test as minimal as possible. I hope that it makes sense!

Reply:
> I hope that it makes sense!

Makes perfect sense!

Unskipped the PyVista test since we now have a virtual frame buffer.

The remaining skip is reported in https://github.com/pyxem/kikuchipy/issues/707 with a plan to fix. This can be unskipped when a fix is pushed.